### PR TITLE
Add emojis to nixpkgs-review post-result

### PR DESCRIPTION
**Pull request by Pandapip1**
Fixes https://github.com/Mic92/nixpkgs-review/issues/404

### CODE REVIEW

File: nixpkgs_review/report.py [hunk 1]
```diff
@@ -180,10 +180,10 @@ def markdown(self, pr: int | None) -> str:
             self.non_existent,
```
Comment (Artturin) [inline nixpkgs_review/report.py:180]: ^ missed those 2

Comment (Pandapip1) [reply nixpkgs_review/report.py:181]: Yup, just fixed.

**Comment by Artturin** [comment]
Formatting fails, do `nix fmt`

**Comment by Pandapip1** [comment]
Done! TIL `nix fmt` exists.

**Review by Artturin — APPROVED** [review]

**Pull request merged**